src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/camunda/Task.java

```diff
@@ -0,0 +1,249 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import java.time.ZonedDateTime;
+import java.util.Objects;
+
+import static uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.CamundaTime.CAMUNDA_DATA_TIME_FORMAT;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@SuppressWarnings({"PMD.LawOfDemeter","PMD.TooManyFields","PMD.ExcessiveParameterList","PMD.ShortClassName"})
```

Comment:
Let's try to address the PMD. Otherwise, what's the point to use it
There are actually too many fields, and the constructor has too many parameters, right? Probably, it's useful to use a builder or to create another POJOs to organise similar semantic fields.